sql/sqlite/SQLite.g4

```diff
@@ -397,7 +398,54 @@ common_table_expression
 result_column
  : '*'
  | table_name '.' '*'
- | expr ( K_AS? column_alias )?
+ | (expr|windowFunction) ( K_AS? column_alias )?
+ | windowFunction 
```

Comment:
This line can be removed since a single `windowFunction` can already be matched by `| (expr|windowFunction) ( K_AS? column_alias )?`

Reply:
Done ,Thanks